src/plugins/cloudinary/models/video-source.js

```diff
@@ -172,6 +172,9 @@ class VideoSource extends BaseSource {
       if (codecTrans && !hasCodecSrcTrans) {
         opts.transformation.push(codecTrans);
       }
+      if (opts.format === 'auto') {
+        delete opts.format;
+      }
```

Comment:
@yaniv-li does that belong here?

Reply:
this is the backend CDN chose best option